scripts/apps/authoring/authoring/services/AuthoringService.js

```diff
@@ -417,9 +417,7 @@ export function AuthoringService($q, $location, api, lock, autosave, confirm, pr
         // item is published state - corrected, published, scheduled, killed
         if (self.isPublished(currentItem)) {
             // if not the last published version
-            if (angular.isDefined(item.archive_item) &&
-                item._current_version !== item.archive_item._current_version &&
-                item.operation !== 'move' && item.archive_item.operation !== 'move') {
+            if (item.last_published_version == false) {
```

Comment:
It should be three equals in the comparison.

Reply:
done

Reply:
Looks good now. I'll approve it once tests pass :)

Reply:
just wondering - is it better to use `=== false` instead of `! item.last_published_version` ?

Reply:
I prefer comparing to `false` since it the allows me to infer that the value is boolean and not, for example, an integer. If it was a property of a TypeScript object, I wouldn't care either way.

Reply:
Any conclusion on this?